molecule/default/verify.yml

```diff
@@ -35,3 +39,41 @@
       msg: |
         Found warnings in output of `apt update`:
         {{ apt_update.stderr }}
+
+  # Testing dokku_clone
+  - name: install rsync
+    apt:
+      name: rsync
```

Comment:
hm... maybe this should be done in the role...
`git:sync` was failing when rsync is not present